Cargo.toml

```diff
@@ -51,9 +51,6 @@ members = [
 
 exclude = ["crates/release-automation"]
 
-[build]
```

Comment:
cargo warned about this as an unused flag. It doesn't show up in the manifest handbook as a flag.